README.md

```diff
@@ -12,17 +12,30 @@ Official project homepage:
 ![jQuery-USOS widgets preview](http://i.imgur.com/jyJy1w5.png)
 
 
+Installation
+------------
+
+To install type: 
+
+* `npm install` 
+
+* Then do `npm run release` to build release in lib/ folder 
+
+* Then you can start static server with `npm run server`
+
+* Go to localhost:3000 and preview your examples! 
```

Comment:
Większość przykładów u mnie nie działa. Czy to coś u mnie nie tak, czy ta zmiana jest nadal "in progress"?

Czy dałoby radę te przykłady umieścić w jsfiddle lub w czymś podobnym, żeby oglądający mógł widzieć i edytować kod?